code/go/0chain.net/mocks/core/datastore/store.go

```diff
@@ -0,0 +1,197 @@
+// Code generated by mockery 2.7.4. DO NOT EDIT.
+
+package mocks
```

Comment:
It smells bad having all sub-packages with the same name.
It strongly suggests to me that the directories should be merged into one package `mocks`. Is there any reason not to collapse the directory structure into `code/go/0chain.net/mocks`?

Reply:
Moved all mocks in a single package.